src/compiler/scala/tools/nsc/typechecker/RefChecks.scala

```diff
@@ -458,9 +458,9 @@ abstract class RefChecks extends Transform {
               // Only warn for the pair that has one leg in `clazz`.
               if (clazz == memberClass) checkOverrideDeprecated()
               if (settings.warnNullaryOverride) {
-                if (other.paramss.isEmpty && !member.paramss.isEmpty && !member.isJavaDefined) {
+                def javaDetermined(sym: Symbol) = sym.isJavaDefined || isUniversalMember(sym)
+                if (other.paramss.isEmpty && !member.paramss.isEmpty && !javaDetermined(member) && !member.overrides.exists(javaDetermined))
```

Comment:
This is what dot[ty|i] does. [I worry a little bit about the compiler efficiency of the `check allOverriddenSymbols.exists(test)`](https://contributors.scala-lang.org/t/nullary-and-nilary-methods/3756/10?u=lrytz) -- @retronym wdyt?

Reply:
I'll keep an eye on the grafana.

Reply:
Looks OK